static/app/views/dashboardsV2/widgetCard/metricsWidgetQueries.tsx

```diff
@@ -114,14 +115,35 @@ class MetricsWidgetQueries extends React.Component<Props, State> {
 
   private _isMounted: boolean = false;
 
-  fetchTabularData(_queryFetchID: symbol) {
-    this.setState({loading: false, tableResults: []});
-    // TODO(dam): implement the rest
+  get limit() {
+    switch (this.props.widget.displayType) {
+      case DisplayType.TOP_N:
+        return TOP_N;
+      case DisplayType.BIG_NUMBER:
```

Comment:
We should limit the table results to 20 (that's what we do for events/issue widgets if i recall correctly)